types/node/events.d.ts

```diff
@@ -670,7 +670,7 @@ declare module 'events' {
             }
         }
     }
-    export = EventEmitter;
+    export default EventEmitter;
```

Comment:
This is not equivalent and would be a breaking change.

Reply:
Indeed, but exporting the three interfaces makes it an error. I'm very far from an expert so if there is a way to make it a non-breaking change, that would be what I'd suggest.